src/request/util.ts

```diff
@@ -8,10 +8,9 @@ import { forOf } from '@dojo/shim/iterator';
  * @param url The base URL.
  * @param options The RequestOptions used to generate the query string or cacheBust.
  */
-export function generateRequestUrl(url: string,
-		{ query, cacheBust }: RequestOptions = {}): string {
-	query = new UrlSearchParams(query).toString();
-	if (cacheBust) {
+export function generateRequestUrl(url: string, options: RequestOptions = {}): string {
+	let query = new UrlSearchParams(options.query).toString();
```

Comment:
Why not just move the destructering down?

```ts
export function generateRequestUrl(url: string, options: RequestOptions = {}): string {
    let { query, cacheBust } = options;
```

Reply:
I changed it for a couple reasons:

- I thought it'd be weird to set `query` and then set it again on the line after it.
```ts
	let { query, cacheBust } = options;
	query = new UrlSearchParams(query).toString();
```
- I didn't want cacheBust to be a `let` variable since it's not modified.

Reply:
Yeah, ok, and it is only accessed once (`cacheBust`) 👍